client/src/App.tsx

```diff
@@ -659,6 +764,7 @@ function App() {
 	return (
 		<div
 			className="app"
+			ref={fireWorkContainer}
```

Comment:
mentioned before but a better name for the ref would be something like appRef, since it's clearer and more accurate